### ISSUE TITLE
How can I ignore this error of unresolved import for an automation test?

### ISSUE THREAD

**Issue by tal-rofe** [webpro/knip#122]
I have the following file in my project, `./automation/main.spec.cjs`:
```js
/**
 * * Testing there is no ESM compiled module in the distribution artifact
 * * As "commitizen" is yet to support ESM adapters, we must not use any ESM-only packages as dependencies
 * * If this "require" statement fails, probably the distribution artifact depends on ESM package
 * * This will cause a "commitizen" failure if it is dependent on ESM package
 */
require('../dist/index.cjs');

process.exit(0);
```

When running `knip` in the project, I get an error:
```
Unresolved imports (1)
../dist/index.cjs  automation/main.spec.cjs
```
Well, yes, `../dist/index.cjs` indeed does not exist. But "I DONT CARE". Is there an option to ignore this error?

**Comment by webpro** [comment]
You can't really suppress such an error, but you can add `automation/main.spec.cjs` to the list of `ignore: []` patterns.

**Comment by tal-rofe** [comment]
> You can't really suppress such an error, but you can add `automation/main.spec.cjs` to the list of `ignore: []` patterns.
I tried, it didn't help. Got same error again

**Comment by webpro** [comment]
Not sure what I can do without seeing the thing. Do you happen to have a public repo or reproducible test case?

**Comment by tal-rofe** [comment]
> Not sure what I can do without seeing the thing. Do you happen to have a public repo or reproducible test case?

yes, here: 
https://github.com/Exlint/cz-vinyl/actions/runs/4772280090/jobs/8484693564

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.10.2. See [Release 2.10.2](https://github.com/webpro/knip/releases/tag/2.10.2) for release notes._

**Comment by webpro** [comment]
The bug was caused by file dependencies resolved from `npm scripts` not going through the `ignore` filter. That should be fixed now.
